src/schema/v2/homeView/HomeViewSection.ts

```diff
@@ -274,6 +277,24 @@ export const HomeViewGalleriesSectionType = new GraphQLObjectType<
   },
 })
 
+export const HomeViewDiscoverMarketingCollectionType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: HomeViewSectionTypeNames.HomeViewSectionDiscoverMarketingCollections,
+  description: "A section containing a list of curated marketing collections",
+  interfaces: [HomeViewGenericSectionInterface, NodeInterface],
+  fields: {
+    ...standardSectionFields,
+    linksConnection: {
```

Comment:
**thought:** I would like to encourage us to think in terms of the reusability of generic section types.

Here we have a section whose standard presentation in the UI will be a set of pills/chips. Each chip represents a link to _something_. 

- Does that something need to be declared as a _marketing collection_ here, as opposed to a more generic group of links or featured links?

- Does the section need to be coupled to the _discover_ language, which seems to me an incidental (and possibly changeable) detail of the current UI?

It makes sense to me that the section declaration is over in `sections/DiscoverMarketingCollections.ts` adopts these more specific details, because that _is a specific instance_ of this section type.

But the language here in the schema strikes me as too specific, too coupled to the language of this particular **Discover Something New** section.

Instead I might envision something like this new GraphQL _section type_:

```graphql
type HomeViewSectionFeaturedLinks {
  # etc…
}
```

and this new _section instance_:


```js
export const DiscoverMarketingCollections: HomeViewSection = {
   id: "home-view-section-discover-marketing-collections",

   type: HomeViewSectionTypeNames.HomeViewSectionFeaturedLinks,
  // etc...
}
```

cc @artsy/onyx-devs for your thought as well

Reply:
(Noting also an alternative approach sketched out in #6087)